index.html

```diff
@@ -1,320 +1,473 @@
-      <ul class="social-icons display-flex justify-content">
-        <li class="icon-frame">
-          <a href="#"><img src="Images/Icon-GitHub.png" alt="Github logo"></a>
-        </li>
-        <li class="icon-frame">
-          <a href="#"><img src="Images/Icon-Linkedin.png" alt="LinkedIn logo"></a>
-        </li>
-        <li class="icon-frame">
-          <a href="#"><img src="Images/Icon-Angelist.png" alt="AAngelist logo"></a>
-        </li>
-        <li class="icon-frame">
-          <a href="#"><img src="Images/Icon-Twitter.png" alt="Twitter logo"></a>
-        </li>
-        <li class="icon-frame">
-          <a href="#"><img src="Images/Icon-Medium.png" alt="Medium logo"></a>
-        </li>
-      </ul>
-    </div>
-    <section class="vector display-flex justify-content style">
-      <div>
-        <img src="Images/Vector-Icon.png" alt="Down Arrow icon">
+    <section>
+      <div class="about-work display-flex" id="cards-section">
+        <h2 class="style">
+          MY RECENT<br />
+          WORK
+        </h2>
       </div>
-      <div>
-        <h3 class="word style">Work</h3>
+    </section>
+    <section class="about-work-div">
+      <div class="work-header display-flex">
+        <h2 class="style">My recent work</h2>
       </div>
     </section>
-  </section>
-  <section>
-    <div class="about-work display-flex" id="cards-section">
-      <h2 class="style">MY RECENT<br> WORK</h2>
-    </div>
-  </section>
-  <section class="about-work-div">
-    <div class="work-header display-flex">
-      <h2 class="style">My recent work</h2>
-    </div>
-  </section>
-  <div class="line3"></div>
-  <main>
-    <section class="work-container">
-      <article class="cards-container display-flex">
-        <div class="placeholder"></div>
-        <div class=" details-div display-flex">
-          <div class="cards-header display-flex">
-            <h3 class="style">Multi-Post Stories</h3>
+    <div class="line3"></div>
+    <main>
+      <!-- <section class="work-container">
+        <article class="cards-container reverse-div display-flex">
+          <div class="placeholder"></div>
+          <div class="details-div display-flex">
+            <div class="cards-header display-flex">
+              <h3 class="style">Multi-Post Stories</h3>
+            </div>
+            <div class="card-info">
+              <p class="style">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+            </div>
+            <ul class="languages-div">
+              <li class="lang style">css</li>
+              <li class="lang style">html</li>
+              <li class="lang style">bootstrap</li>
+              <li class="lang style ruby">Ruby</li>
+            </ul>
+            <div class="btn-div">
```

Comment:
- [ ] Clean up this file. It is a better practice to remove all chunks of commented-out code from the repo

Reply:
Thank you for this review, I will clean it up and make my code clean.